components/request-card/utils.js

```
@@ -0,0 +1,457 @@
+  if (isExpanded) {
+    accordion.classList.remove('active');
+    panel.style.maxHeight = null;
+  } else {
+    document.querySelectorAll('.accordion.active').forEach((el) => {
+      el.classList.remove('active');
+      el.nextElementSibling?.style &&
+        (el.nextElementSibling.style.maxHeight = null);
+    });
+
+    accordion.classList.add('active');
+    panel.style.maxHeight = panel.scrollHeight + 'px';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)